docs/advanced/migrate_to_0.8.0.md

```diff
@@ -30,17 +28,21 @@ gem "decidim", "~> 0.8.0"
 gem "decidim-verifications"
 ...
 ```
+
 ### bundle update
-Finally run *bundle update* to get the required gems updated.
+
+Finally run _bundle update_ to get the required gems updated.
```

Comment:
Use backticks instead?

Reply:
Totally, this was automatically changed.